expts/main_run_multitask.py

```diff
@@ -115,4 +119,9 @@ def main(cfg: dict, run_name: str = "main", add_date_time: bool = True) -> None:
         CONFIG_FILE = args.config
     cfg = load_yaml_config(CONFIG_FILE, MAIN_DIR, unknown_args)
 
-    main(cfg)
+    for num_workers in [4, 8, 16, 24]:
+        for batch_size in [128, 256, 384, 512]:
```

Comment:
What's the intuition for this bit? Is it for GPU optimization?

Reply:
It was for testing purposes only. Will be removed in the next commit.